cabal.project

```diff
@@ -54,6 +54,21 @@ test-show-details: direct
 -- Always write GHC env files, because they are needed for ghci.
 write-ghc-environment-files: always
 
+constraints:
+  , plutus-ledger-api ^>= 1.31
+  , prettyprinter-configurable ^>= 1.31
+
+allow-newer:
+  -- These are required due to the above.
+  , cardano-ledger-alonzo:plutus-ledger-api
+  , cardano-ledger-alonzo-test:plutus-ledger-api
+  , cardano-ledger-babbage:plutus-ledger-api
+  , cardano-ledger-binary:plutus-ledger-api
+  , cardano-ledger-conway:plutus-ledger-api
+  , cardano-api:plutus-ledger-api
+  , cardano-api:prettyprinter-configurable
+  , cardano-api:base
+
```

Comment:
I will let the node team decide whether the above needs to be fixed before this can be merged.